docs/installation/cloud/cloud-ex-aws.md

```diff
@@ -197,8 +197,12 @@ For Ubuntu Trusty (and some other versions), it’s recommended to install the `
 
 ## Where to go next
 
-* Would you like a quicker way to do Docker cloud installs? See [Digital Ocean Example: Use Docker Machine to provision Docker on cloud hosts](cloud-ex-aws.md).
+* Looking for a quicker way to do Docker cloud installs and provision multiple hosts? See these topics on using <a href="https://docs.docker.com/machine/overview/" target="_blank">Docker Machine</a> for provisioning.
```

Comment:
do we want these links to open a new window? given that this is the end of the page, I think navigating away from it without opening a new window should be okay